src/jdk.compiler/share/classes/com/sun/tools/javac/code/Source.java

```diff
@@ -207,15 +207,9 @@ public Target requiredTarget() {
      */
     public enum Feature {
 
-        DIAMOND(MIN, Fragments.FeatureDiamond, DiagKind.NORMAL), // Used in Analyzer
         MODULES(JDK9, Fragments.FeatureModules, DiagKind.PLURAL),
         EFFECTIVELY_FINAL_VARIABLES_IN_TRY_WITH_RESOURCES(JDK9, Fragments.FeatureVarInTryWithResources, DiagKind.PLURAL),
         DEPRECATION_ON_IMPORT(MIN, JDK8),
-        POLY(JDK8),
-        LAMBDA(JDK8, Fragments.FeatureLambda, DiagKind.PLURAL),
-        DEFAULT_METHODS(JDK8, Fragments.FeatureDefaultMethods, DiagKind.PLURAL),
```

Comment:
As noted in https://github.com/openjdk/jdk/pull/10074#issuecomment-1230998246, if DEFAULT_METHODS is removed, there is a core libs test that will fail.

I believe the functionality request is a standard idiom for forcing a source file to be compiled as Preview. Merely setting -XDpreview (however that hidden option is spelled) is not sufficient. A feature that triggers a checkSourceLevel call is needed too.

Reply:
that can be worked out I think as I did with other tests, it should show up in the mach5 run. Do u remember the test name?

Reply:
Pretty sure the test in question was under java.lang.invoke in the libraries (i.e. test/jdk vs test/langtools) area.

If we come up for a supported idiom for this, there is at least one other test I'd want to update to use it; that can be done as future work.

Reply:
all tests are passing in mach5

Reply:
Sounds good.

Reply:
thanks for the review